electron/app/components/Player51.tsx

```diff
@@ -11,19 +11,20 @@ const PARSERS = {
     (name, obj) => {
       return {
         type: "eta.core.data.CategoricalAttribute",
-        name: name,
+        name,
         confidence: obj.confidence,
         value: obj.label,
       };
     },
   ],
   Detection: [
     "objects",
-    (g, obj) => {
+    (name, obj) => {
       const bb = obj.bounding_box;
       return {
         type: "eta.core.objects.DetectedObject",
-        label: `${g}:${obj.label}`,
+        name,
+        label: `${obj.label}`,
```

Comment:
This leads to issues where multiple detections with the same label but different names will have the same color, since this field is currently also used to determine the color. Rolling back fixes the issue: https://github.com/voxel51/fiftyone/commit/8cfd3e28a432569ba8bd095248689fc0092a5401
(linking here too for reference)